erpnext/manufacturing/doctype/work_order/work_order.py

```diff
@@ -241,7 +241,13 @@ def on_submit(self):
 		if not self.fg_warehouse:
 			frappe.throw(_("For Warehouse is required before Submit"))
 
-		self.update_work_order_qty_in_so()
+		prod_plan = frappe.get_doc('Production Plan', self.production_plan)
+		
+		if prod_plan.prod_plan_ref:
```

Comment:
if self.production_plan and frappe.db.exists('Production Plan Item Reference', {'parent': self.production_plan})